scripts/fuzz_opt.py

```diff
@@ -145,7 +145,11 @@ def randomize_feature_opts():
         # 2/3 of the remaining 90% use them all. This is useful to maximize
         # coverage, as enabling more features enables more optimizations and
         # code paths, and also allows all initial contents to run.
-        pass
+
+        # The shared-everything feature is new and we want to fuzz it, but it
+        # also currently disables fuzzing V8, so disable it half the time.
+        if random.random() < 0.5:
```

Comment:
Skipping V8 is pretty significant, how about 0.9 here?

Reply:
Sure, I guess in my local fuzzing I can always comment this out.